docs/book/src/cronjob-tutorial/testdata/project/test/e2e/e2e_test.go

```diff
@@ -84,31 +96,161 @@ var _ = Describe("controller", Ordered, func() {
 				)
 
 				podOutput, err := utils.Run(cmd)
-				ExpectWithOffset(2, err).NotTo(HaveOccurred(), "Failed to retrieve controller-manager pod information")
+				g.Expect(err).NotTo(HaveOccurred(), "Failed to retrieve controller-manager pod information")
 				podNames := utils.GetNonEmptyLines(string(podOutput))
-				if len(podNames) != 1 {
-					return fmt.Errorf("expected 1 controller pod running, but got %d", len(podNames))
-				}
+				g.Expect(podNames).To(HaveLen(1), "expected 1 controller pod running")
 				controllerPodName = podNames[0]
-				ExpectWithOffset(2, controllerPodName).Should(ContainSubstring("controller-manager"))
+				g.Expect(controllerPodName).To(ContainSubstring("controller-manager"))
 
 				// Validate the pod's status
 				cmd = exec.Command("kubectl", "get",
 					"pods", controllerPodName, "-o", "jsonpath={.status.phase}",
 					"-n", namespace,
 				)
-				status, err := utils.Run(cmd)
-				ExpectWithOffset(2, err).NotTo(HaveOccurred(), "Failed to retrieve controller-manager pod status")
-				if string(status) != "Running" {
-					return fmt.Errorf("controller pod in %s status", status)
-				}
-				return nil
+				g.Expect(utils.Run(cmd)).To(BeEquivalentTo("Running"), "Incorrect controller-manager pod status")
 			}
 			// Repeatedly check if the controller-manager pod is running until it succeeds or times out.
-			EventuallyWithOffset(1, verifyControllerUp, time.Minute, time.Second).Should(Succeed())
+			Eventually(verifyControllerUp, time.Minute, time.Second).Should(Succeed())
```

Comment:
```suggestion
			Eventually(verifyControllerUp, time.Minute, 20* time.Second).Should(Succeed())
```

Could we test each 20 or 10 seconds?

Reply:
Oh, my bad, I botched this syntax. It should have been 
```suggestion
			Eventually(verifyControllerUp).WithTimeout(time.Minute).WithInterval(time.Second).Should(Succeed())
```

The default interval is 100ms, I think, so that's why the log was filled with these.

I think I can configure the timeout and intervals at the top of the test instead.  No need to litter the code with this.